src/luos.c

```diff
@@ -75,8 +75,10 @@ void Luos_Loop(void)
         // check if this is a Luos Command
         uint8_t cmd = 0;
         uint16_t size = 0;
-        LUOS_ASSERT(MsgAlloc_GetLuosTaskCmd(remaining_msg_number, &cmd) == SUCCEED);
-        LUOS_ASSERT(MsgAlloc_GetLuosTaskSize(remaining_msg_number, &size) == SUCCEED);
+        if((MsgAlloc_GetLuosTaskCmd(remaining_msg_number, &cmd) != SUCCEED)||(MsgAlloc_GetLuosTaskSize(remaining_msg_number, &size) != SUCCEED))
+        {
+            break;
+        }
```

Comment:
I'm not sure to agree here.
We have massage and get cmd and size. If we fail we should assert.
I can't see any way to fall in this case that not represent a critical failure.
But the previous code was wrong because if we disable the assert functions will be not executed...

Reply:
done